autodist/cluster.py

```diff
@@ -135,6 +138,11 @@ def get_local_address(self):
         Returns:
             str: Worker ip or chief address by default.
         """
+        if IS_ADAPTDL:
+            hostname = socket.gethostname()
+            local_ip = socket.gethostbyname(hostname)
+            return local_ip 
```

Comment:
Since there is already a class named `ADAPTDLCluster` inherited from `Cluster`, is it necessary to insert ADAPTDL related code in the base class?

Reply:
Thanks for pointing this out. I have updated it to the ADAPTDLCluster Class. Thanks!